### DISCUSSION TITLE
Issue with parsing nested html tags

### DISCUSSION THREAD

**Discussion by avinashjain750** [jhy/jsoup#2138] [Q&A]
I have been having issues using jsoup html parser. For some reason, it is not recognizing the nested html tags, taking only the first one. Any inputs would be greatly appreciated.

**Comment by jhy** [comment]
Can you provide a test case that shows what you are doing and what you expect?

jsoup very definitely supports nested HTML. It wouldn't be much of an HTML parser if it didn't...
